fe/fe-core/src/main/java/com/starrocks/analysis/LambdaFunction.java

```diff
@@ -0,0 +1,63 @@
+package com.starrocks.analysis;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.sql.ast.AstVisitor;
+import com.starrocks.sql.common.ErrorType;
+import com.starrocks.sql.common.StarRocksPlannerException;
+import com.starrocks.thrift.TExprNode;
+import com.starrocks.thrift.TExprNodeType;
+
+import java.util.List;
+
+public class LambdaFunction extends Expr {
```

Comment:
I suggest calling it LambdaExpression, because SR has a xxxFunction system (TableFunction, ScalarFunction, ...) in com.starrocks.catalog. LambdaFunction can be misleading. And this class itself is also a class under the Expression system

Reply:
rename to `LambdaFunctionExpr`